jujugui/static/gui/src/app/models/models.js

```diff
@@ -1505,14 +1505,14 @@ YUI.add('juju-models', function(Y) {
         return {
           parentId: null,
           containerType: null,
-          number: parseInt(name, 10)
+          number: name
```

Comment:
The machine/container ids now need to be strings so that they work with uncommitted ids e.g. 'new0'. I don't think this code has been used in a long while.